www/js/lib/popovers.js

```diff
@@ -104,14 +104,12 @@ function cleanUpLedeContent (node) {
     styleElements.forEach(style => {
         style.parentNode.removeChild(style);
     });
-    const paragraphs = Array.from(node.querySelectorAll('p'));
-    // Filter out empty paragraphs or those with less than 50 characters
+    const paragraphs = Array.from(node.querySelectorAll('p:not(#pcs-edit-section-title-description)'));
```

Comment:
This is good, but I think we should generalize this function, as in the future we may want to exclude other elements by css selector. See overall comment.